setup/module/api/load_balancer.tf

```diff
@@ -65,7 +65,7 @@ resource "aws_elb" "api" {
     healthy_threshold   = 2
     unhealthy_threshold = 2
     timeout             = 5
-    target              = "HTTP:80/admin/health"
```

Comment:
Why is this being changed?

Reply:
The `admin/health` endpoint requires auth. It's easier to change this than to have one-off routes in `api`

Reply:
I opened issue #456 to address this change.